extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/support/FastCamelContext.java

```diff
@@ -365,11 +388,12 @@ public AsyncProcessor createMulticast(Collection<Processor> processors, Executor
     @SuppressWarnings("unchecked")
     protected <T> T resolve(Class<T> clazz, String type, String name, CamelContext context) {
```

Comment:
What is this method used for? Can we add some javadoc, and the code that is disabled is it to be removed or not? Maybe add a TODO if this code needs revisit

Reply:
it was originally used to auto configure component, languages and data formats but it is not more needed now as MainSupport does it, let me revisit it

Reply:
done